src/utils/Database/LSMTree/Util/Orphans.hs

```diff
@@ -55,15 +57,16 @@ deriving anyclass instance Uniform Word256
 instance Hashable Word256 where
   hashIO32 (Word256 a b c d) = hashIO32 (a, b, c, d)
 
--- | Placeholder instance, not optimised
 instance SerialiseKey Word256 where
   serialiseKey (Word256{word256hi, word256m1, word256m0, word256lo}) =
-      serialiseKey $ B.toLazyByteString $ mconcat [
-          B.word64BE word256hi
-        , B.word64BE word256m1
-        , B.word64BE word256m0
-        , B.word64BE word256lo
-        ]
+    RB.RawBytes $ PV.Vector 0 32 $ P.runByteArray $ do
+      ba <- P.newByteArray 32
+      P.writeByteArray ba 0 $ byteSwap64 word256hi
+      P.writeByteArray ba 0 $ byteSwap64 word256m1
+      P.writeByteArray ba 0 $ byteSwap64 word256m0
+      P.writeByteArray ba 0 $ byteSwap64 word256lo
```

Comment:
```suggestion
      P.writeByteArray ba 0 $ byteSwap64 word256hi
      P.writeByteArray ba 1 $ byteSwap64 word256m1
      P.writeByteArray ba 2 $ byteSwap64 word256m0
      P.writeByteArray ba 3 $ byteSwap64 word256lo
```
?

Reply:
Too passive aggressive, :-1:

Reply:
Ah, I'm sorry. It wasn't intended like that. I have made similar mistakes in the past that became clear the moment I looked at it again, so it didn't seem like writing a longer comment would add much.

Reply:
No worries, I noticed that right away myself as tests failed, so it came like salting a fresh wound.